File: src/SQL8r/Doc/TestCode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Infragistics.Win.UltraWinTree;

namespace SQL8r.WinForms.NodeTypes.Project
{
    public class FileNode : UltraTreeNode
    {
        public FileNode(string name)
        {
            base.Text = name;
        }

        public string FileName
        {
            get { return base.Text; }
        }
    }
}

File: src/SQL8r/Doc/TestCode.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports Infragistics.Win.UltraWinTree

Namespace SQL8r.WinForms.NodeTypes.Project

    Public Class FileNode
        Inherits UltraTreeNode



        Public Function GetFileName() As String

            Return base.Text
        End Function

    End Class
End Namespace
